src/useWindowSize/__tests__/dom.ts

```diff
@@ -25,34 +25,29 @@ describe('useWindowSize', () => {
 	});
 
 	it('should render', () => {
-		const { result } = renderHook(() => useWindowSize());
-		expect(result.error).toBeUndefined();
+		expect(() => {
+			renderHook(() => useWindowSize());
+		}).not.toThrow();
 	});
 
 	it('should use provided state hook', () => {
 		const { result } = renderHook(() => useWindowSize(useState));
 
 		expect(result.current.width).toBe(100);
 		expect(result.current.height).toBe(100);
-		expect(result.all.length).toBe(1);
 
-		triggerResize('width', 200);
+		triggerResize('height', 200);
```

Comment:
@xobotyi  Why did we changed it to `hieght` from `width`?

Reply:
I have no idea, to be fair =) it is not intended

There is still an issue with harness - have to rewrite it